common/app/model/content/Atoms.scala

```diff
@@ -53,6 +54,11 @@ final case class MediaAtom(
       "^0".r.replaceFirstIn(formattedDuration, "") //strip leading zero
     }
   }
+
+  def activeAsset: MediaAsset = activeVersion match {
+    case Some(version) => assets.find(_.version == version).getOrElse(assets.head)
+    case _ => assets.head
+  }
```

Comment:
It would be better to make `activeAsset` return an `Option[MediaAsset]` and deal with it in the controller or templates.